docs/excel/custom-functions-best-practices.md

```diff
@@ -29,7 +29,19 @@ function getComment(x) {
 ```
 
 ## Debugging
-Currently, the best method for debugging Excel custom functions is to first [sideload](../testing/sideload-office-add-ins-for-testing.md) your add-in within **Excel Online**. You can then debug your custom functions by using the [F12 debugging tool native to your browser](../testing/debug-add-ins-in-office-online.md). Use `console.log` statements within your custom functions code to send output to the console in real time.
+
+Currently, the best method for debugging Excel custom functions is to first [sideload](../testing/sideload-office-add-ins-for-testing.md) your add-in within **Excel Online**. You can then debug your custom functions by using the [F12 debugging tool native to your browser](../testing/debug-add-ins-in-office-online.md) in combination with the following techniques:
```

Comment:
Since sideloading now happens automatically with either npm start on desktop or npm run start-web with online should we link to that article? I feel like we might just want to say make sure you have run one of those commands and then debug. Unless we want the functions to be sideloaded and registered? It seems like the content in the sideload article is what we call registering in the tutorial?

Reply:
Gotcha! :)

Reply:
Additionally, keep in mind that the `npm run start-web` and `npm start` commands are specific to projects that are created with Yo Office (those commands are defined in the **package.json** file for the project that Yo Office provides). If a developer creates their own project from scratch (i.e., doesn't use Yo Office), then we can't make any assumptions about what these commands might do. Since this is the "best practices" article, we should be providing best practices that apply to a custom functions project regardless of how it's created (i.e., using Yo Office or otherwise).